cmd/validate/image_test.go

```diff
@@ -673,6 +674,104 @@ configuration:
 	assert.NoError(t, err)
 }
 
+func Test_ValidateImageCommandExtraData(t *testing.T) {
+	validate := func(_ context.Context, component app.SnapshotComponent, _ policy.Policy, _ bool) (*output.Output, error) {
```

Comment:
There is a compile error on this line:
```suggestion
	validate := func(_ context.Context, component app.SnapshotComponent, _ policy.Policy, _ []evaluator.Evaluator, _ bool) (*output.Output, error) {
```

Reply:
It looks like @mbestavros did a rebase that brought in the change in the signature for the validate function. I have questions...

- The only CI check failing right now is the linter. How did "Checks / Test" pass?!
- Why was the linter complaining about this line before when it was obviously correct at the time?